### ISSUE TITLE
official aws sdk support

### ISSUE THREAD

**Issue by nuvanti** [softprops/dynomite#162]
Does dynomite work with the official aws rust sdk (https://github.com/awslabs/aws-sdk-rust) ? 

If not, do you have any plans?

**Comment by ZaMaZaN4iK** [comment]
No, this library doesn't support the official AWS SDK (at least yet). Consider that for now the official Rust AWS SDK is not considered stable and at least from AWS perspective shouldn't be used in production yet.

We can wait for some time and when migrate from rusoto to AWS Rust SDK. However, later I'm going to add the official AWS Rust SDK support to the library here: https://github.com/rust-serverless/dynomite since this fork is not maintained now.

**Comment by softprops** [comment]
I did have plans to migrate this at some point to the official sdk. It would actually simplify and reduce the tax of maintenance as the sdk now fills some feature gaps in rusoto that no longer need to be dynomite specific

**Comment by nuvanti** [comment]
Thanks!

**Comment by pstephens** [comment]
The AWS SDK is now published to crates.io, so GA is getting closer! https://crates.io/crates/aws-sdk-dynamodb

Would both rusoto and aws-sdk back ends be available via feature flags? Or would this be a hard cut over (to reduce tech debt)?

**Comment by sdd** [comment]
In case anyone is interested (and bear in mind this is completely unsupported and unstable right now) I've been working with a fork of https://github.com/rust-serverless/dynomite that I've refactored from `rusoto` to `aws-sdk-rust`, at https://github.com/sdd/dynomite/tree/aws-sdk-dynamodb-support .  Tests and Doctests are working. Comments and contributions encouraged, and happy to work with @softprops or @nuvanti when it comes to the time to do this for real in your repos.

**Comment by stanislav-tkach** [comment]
@sdd Could you please update your fork to use the latest AWS SDK version? I'm trying to rewrite my code that uses rusoto to the official SDK and an updated `dynomite` would be very handy.

**Comment by sdd** [comment]
@stanislav-tkach Sure, I'll take a look later this evening 👍🏼

**Comment by stanislav-tkach** [comment]
@sdd Thanks!

**Comment by zeeshanabid94** [comment]
Is this issue being actively worked on? I know @sdd already has a fork with `aws-sdk`. Is there a plan to get that merged in?
